src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1923,7 +1939,10 @@ def get_terms(cmd, urn=None, publisher=None, offer=None, plan=None):
     """
     publisher, offer, plan = _terms_prepare(cmd, urn, publisher, offer, plan)
     op = cf_vm_image_term(cmd.cli_ctx, '')
-    terms = op.get(publisher, offer, plan)
+    terms = op.get(offer_type='virtualmachine',
```

Comment:
Can we use `OfferType.VIRTUALMACHINE` here?

Reply:
done